app/components/common/error/index.js

```diff
@@ -0,0 +1,95 @@
+'use strict';
+
+angular.module('odca.error', [])
+  .config(['$httpProvider', function ($httpProvider) {
+    $httpProvider.interceptors.push('errorHttpInterceptor');
+  }])
+  .run(['$window', 'error', function ($window, error) {
+    $window.onerror = onError;
```

Comment:
We should be able to avoid the global here. And if we can't, we should give it an odca prefix just in case.